src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1130,6 +1130,7 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
 
                 case SymbolKind.ErrorType:
                 case SymbolKind.NamedType:
+                    // PROTOTYPE emit references to extension types
```

Comment:
>// PROTOTYPE emit references to extension types [](http://example.com/codeflow?start=20&length=47)

I think the comment should go into the method that does translation. This is just a dispatch method.